src/types/simulation.rs

```diff
@@ -55,7 +54,6 @@ pub struct StakeInfo {
 pub enum SimulateValidationError<M: Middleware> {
     UserOperationRejected { message: String },
     OpcodeValidation { entity: String, opcode: String },
-    UserOperationExecution { message: String },
```

Comment:
@Vid201 I don't think this is needed now. But I need your confirmation on it.

Reply:
It is needed for this test: https://github.com/eth-infinitism/bundler-spec-tests/blob/master/tests/rpc/test_eth_estimateUserOperationGas.py#L23.

I suggest to setup local environment for running tests, so that these changes won't break the bundler spec tests (https://github.com/eth-infinitism/bundler-spec-tests).

Reply:
Just start local geth, run bundler (`run-bundler-debug`), and deploy entry point. Then you can run tests from bundler-spec-tests repo: `pdm run pytest -rA -W ignore::DeprecationWarning --url http://127.0.0.1:3000/ --entry-point 0x0576a174D229E3cFA37253523E645A78A0C91B57 --ethereum-node http://127.0.0.1:8545/`

Reply:
Makes sense. I went through the official spec and didn't find it. It turns out to be in bundler-spec-tests.